### PR TITLE
[New] Add egjs-agent@2.1.1 w/ npm auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
### Memo
The origin lib is https://github.com/naver/egjs which host on CDNJS
It's migrating From egjs 1.x.x to egjs 2.0.0 https://github.com/naver/egjs/wiki/Migrating-From-egjs-1.x.x-to-egjs-2.0.0
So I add these libs

---

Pull request for issue: #11777 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/naver/egjs-agent
 * Official website (optional, not the repository): https://naver.github.io/egjs-agent/
 * NPM package url (optional): https://www.npmjs.com/package/@egjs/agent
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 8
   - Count of forks: 2
 * NPM download stats (optional):
   - Downloads in the last day: 6
   - Downloads in the last week: 10
   - Downloads in the last month: 160

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/egjs-agent/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "egjs-agent",
+  "filename": "agent.min.js",
+  "version": "2.1.1",
+  "description": "Extracts browser and operating system information from the user agent string.",
+  "author": "NAVER Corp.",
+  "license": "MIT",
+  "keywords": [
+    "egjs",
+    "agent"
```
Comment (PeterDaveHello) [inline ajax/libs/egjs-agent/package.json]: Where are those keywords from? The agent is meaningless here, use `user agent` or `user`, `agent` instead, maybe even `browser` and `operating system`

Comment (sufuf3) [reply ajax/libs/egjs-agent/package.json]: Keyword from GitHub repo blue part (topics: categorize the repository)

Thank you.

Comment (sufuf3) [reply ajax/libs/egjs-agent/package.json]: @happyhj @sculove could you give us some keywords about this lib?
It's will be useful to us, and let this lib better to be search.
Thank you.

Comment (PeterDaveHello) [reply ajax/libs/egjs-agent/package.json]: I've mentioned the keywords we can use here, just add them please.

**Comment by sufuf3** [comment]
@cdnjs/intern3 I updated keywords.
Please help me review this PR, thank you.

**Pull request merged**